src/main/java/io/github/syst3ms/skriptparser/util/StringUtils.java

```diff
@@ -103,16 +103,6 @@ public static int nextSimpleCharacterIndex(String s, int index) {
                 if (closing == -1)
                     return -1;
                 i = closing;
-            } else if (c == '\'') {
-                var closing = s.indexOf('\'', i + 1);
-                if (closing == -1)
-                    return -1;
-                i = closing;
```

Comment:
Why removing this ? it's single quotes support for nextSimpleCaracterIndex method

Reply:
I removed support for single-quoted strings. Their behavior is now replaced with the raw string syntax (like `r"This is as raw string"`). Not sure why we'd still support this as it can possible cause issues.

Reply:
Reverted the changes as the new syntax did not seem very user-friendly in my opinion.